.env.example

```diff
@@ -0,0 +1,9 @@
+# Example Configuration file
+
+###### GLOBAL VARS #####
+API_URL=https://api.giphy.com/v1/
+
+
+###### IMPORTANT KEYS ######
+YOUR_API_KEY=BtpjCc5CrNS0nxVvRcC8PDz2u3KeW8Bq
```

Comment:
I guess this is not your actual key, right?

Reply:
Yep, I didnt know if you need it so I just copy on .env
Should I removed

Reply:
Yeah remove it, with the name variable is enough to know we should use the key.

Reply:
Done